src/backend/test.rs

```diff
@@ -177,6 +177,10 @@ impl Backend for TestBackend {
         Ok(Rect::new(0, 0, self.width, self.height))
     }
 
+    fn window_size(&mut self) -> Result<(Rect, (u16, u16)), io::Error> {
+        Ok((Rect::new(0, 0, self.width, self.height), (7, 12)))
```

Comment:
What's the significance of 7 and 12 here? Can you document it?

Reply:
It was from when it was `font_size`, not `window_size` - my bad.
Updated to a better value and added a comment.